cmd/workflow-controller/main.go

```diff
@@ -35,21 +35,22 @@ const (
 // NewRootCommand returns an new instance of the workflow-controller main entrypoint
 func NewRootCommand() *cobra.Command {
 	var (
-		clientConfig             clientcmd.ClientConfig
-		configMap                string // --configmap
-		executorImage            string // --executor-image
-		executorImagePullPolicy  string // --executor-image-pull-policy
-		containerRuntimeExecutor string
-		logLevel                 string // --loglevel
-		glogLevel                int    // --gloglevel
-		workflowWorkers          int    // --workflow-workers
-		workflowTTLWorkers       int    // --workflow-ttl-workers
-		podWorkers               int    // --pod-workers
-		podCleanupWorkers        int    // --pod-cleanup-workers
-		burst                    int
-		qps                      float32
-		namespaced               bool   // --namespaced
-		managedNamespace         string // --managed-namespace
+		clientConfig                  clientcmd.ClientConfig
+		configMap                     string // --configmap
+		executorImage                 string // --executor-image
+		executorImagePullPolicy       string // --executor-image-pull-policy
+		containerRuntimeExecutor      string
+		logLevel                      string // --loglevel
+		glogLevel                     int    // --gloglevel
+		workflowWorkers               int    // --workflow-workers
+		workflowTTLWorkers            int    // --workflow-ttl-workers
+		podWorkers                    int    // --pod-workers
+		podCleanupWorkers             int    // --pod-cleanup-workers
+		burst                         int
+		qps                           float32
+		leaderElectionHealthCheckPort string // --leader-election-health-check-port
```

Comment:
is there any preference for string? can you use int?

Reply:
Sure. Updated! Thanks.